pkg/core/server/http.go

```diff
@@ -75,6 +75,8 @@ func (s *Server) startEchoServer() error {
 		g.GET("/graphiql", graphiqlHandler)
 	}
 
+	g.GET("/rewards", s.rewards.GetRewards)
+	g.GET("/rewards/attest", s.rewards.AttestReward)
```

Comment:
Curious why we chose a GET route for this. What do you think about a POST?

Also not sure we care about REST but "attestation" would be the noun, eg. "getAttestation" makes sense